puppet

```diff
@@ -2,5 +2,5 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd $DIR
-bundle exec ./puppetwash.rb $@
+bundle exec ./puppet.rb $@
```

Comment:
Might want to pickup the fix I made for paths with spaces (https://github.com/puppetlabs/boltwash/commit/d9189a20274b8f879b05d71790b7d2ab59407cb3#diff-a6fe881cecd3fb7660083aea35cce430)
```suggestion
bundle exec ./puppet.rb "$@"
```